src/calendar/Calendar.js

```diff
@@ -206,6 +211,33 @@ class Calendar extends Component {
     dbRef.off("value", this.handleDataCallback)
   }
 
+
+  onEventDoubleClick = (eventData)=>{
+    //console.log("testing")
+    //console.log(eventData);
+    // console.log(Object.keys(eventData));
+    // console.log(eventData.e.data.start.value)
+    // console.log(eventData.e.data.end.value)
+    //console.log(this.state.eventClicked)
+
+    let state = this.state;
+    state.eventClicked=true;
+    this.setState(state);
+
+    //console.log("here1")
+    //console.log(state)
+   }
+
+  /**
+   * callback function for EventInvites. Called when the popup window closes
+   * @link EventInvites
+   */
+  eventInviteOnCloseCallback = ()=>{
+    let state = this.state;
+    state.eventClicked=false;
+    this.setState(state);
+  };
```

Comment:
I think mutating state directly is dangerous, id recommend `this.setState({eventClicked: false})`